app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.scss

```diff
@@ -20,6 +20,16 @@
   min-height: 30px;
   box-sizing: border-box;
   background-color: $COLOR--white-two;
+  border: 1px solid $COLOR--gray-80;
+
+  &:hover {
+    border-color: $COLOR--gray-47;
+  }
+  &:focus-within {
```

Comment:
`:focus-within` not supported by ie11

Reply:
Decided to leave it as is (without IE support).
Discussed with @VolhaKarenko.